lib/jelix/plugins/formwidget/date_html/date_html.formwidget.php

```diff
@@ -40,13 +40,13 @@ protected function outputJs() {
         if($maxDate)
             $js .= "c.maxDate = '".$maxDate->toString(jDateTime::DB_DFORMAT)."';\n";
 
+        $this->parentWidget->addJs($js);
+        $this->commonJs();
+
         if($ctrl instanceof jFormsControlDate || get_class($ctrl->datatype) == 'jDatatypeDate' || get_class($ctrl->datatype) == 'jDatatypeLocaleDate'){
             $config = isset($ctrl->datepickerConfig)?$ctrl->datepickerConfig:jApp::config()->forms['datepicker'];
             $js .= 'jelix_datepicker_'.$config."(c, jFormsJQ.config);\n";
         }
-
-        $this->parentWidget->addJs($js);
-        $this->commonJs();
```

Comment:
You cannot remove this code from here, else the call of  jelix_datepicker_* (added just in the 'if' above) will not be done : it will be missing from the $js variable.